src/clipboard_handlers/tables_clipboard.ts

```diff
@@ -69,10 +70,20 @@ export class TableClipboardHandler extends AbstractCellClipboardHandler<
           zones.some((z) => isZoneInside(tableZone, z))
         ) {
           copiedTablesIds.add(table.id);
+          let { numberOfCols, numberOfRows } = zoneToDimension(tableZone);
+          for (let r = tableZone.top; r <= tableZone.bottom; r++) {
+            if (
+              !isCutOperation &&
+              (this.getters.isRowFiltered(sheetId, r) || !rowsIndexes.includes(r))
```

Comment:
`this.getters.isRowFiltered(sheetId, r)` condition is useless no ? filtered rows aren't in rowIndexes